index.html

```diff
@@ -57,6 +55,179 @@ <h2 class="sub-header">I'm a Software Developer</h2>
+                            has been the industry's standard
+                        </p>
+                        <ul class="project-x-list">
+                            <li class="proj-list">HTML</li>
+                            <li class="proj-list">Bootstrap</li>
+                            <li class="proj-list">Ruby</li>
+                        </ul>
+                        <button class="project-btn button">See Projects</button>
+                    </div>
+                    <div class="project-x">
+                        <h1 class="project-x-title">Profesional Art Printing Data</h1>
+                        <p class="project-x-para">
+                            A daily selection of privately personalized reads; 
+                            no accounts or sign-ups required. 
+                            has been the industry's standard
+                        </p>
+                        <ul class="project-x-list">
+                            <li class="proj-list">HTML</li>
+                            <li class="proj-list">Bootstrap</li>
+                            <li class="proj-list">Ruby</li>
+                        </ul>
+                        <button class="project-btn button">See Projects</button>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="about-me">
+            <div class="about-container">
+                <div class="about-me-1">
+                    <h1 class="about-me-title">About Me</h1>
+                    <p class="about-me-para">
+                        Hello I'm a software developer! 
+                        I can help you build a product, 
+                        feature or website Look 
+                        through some of my work and experience! 
+                        If you like what you see and have a project 
+                        you need coded, don't hestiate to contact me.
+                    </p>
+                    <button class="about-me-btn button" >Get My Resume</button>
+                </div>
+                <span class="line-container">
+                    <hr class="line-1">
+                </span>
+                <div class="about-me-common">
+                    <div>
+                        <div class="about-start">
+                            <img src="images/Rectangle 55.svg"/>
+                            <h1 class="about-title">Language</h1>
+                        </div>
+                        <ul class="about-my-list">
+                            <li class="about-list">Javascript</li>
+                            <li class="about-list">Ruby</li>
+                            <li class="about-list">Html</li>
+                            <li class="about-list">Css</li>
+                        </ul>
+                    </div>
+                    <div>
+                        <div class="about-start">
+                            <img src="images/Rectangle 56.svg"/>
```

Comment:
- [ ] Please an `alt` tag to all you images and text in to describe the image because the screen readers rely on that. 🤓 🛠️